pkg/storageincentives/metrics.go

```diff
@@ -18,6 +19,17 @@ type metrics struct {
 	NeighborhoodSelected prometheus.Counter
 	SampleDuration       prometheus.Gauge
 	Round                prometheus.Gauge
+
+	// total calls to chain backend
+	BackendCalls  prometheus.Counter
+	BackendErrors prometheus.Counter
```

Comment:
Ideally these should be exposed by `ChainBackend` interface (or any other type wrapping this interface in order to provide this this functionality.